source/MaterialXRenderGlsl/GlslMaterial.h

```diff
@@ -18,24 +19,12 @@ MATERIALX_NAMESPACE_BEGIN
 
 using GlslMaterialPtr = std::shared_ptr<class GlslMaterial>;
 
-/// @class ShadowState
-/// Helper class representing shadow rendering state
-class MX_RENDERGLSL_API ShadowState
-{
-  public:
-    ImagePtr shadowMap;
-    Matrix44 shadowMatrix;
-    ImagePtr ambientOcclusionMap;
-    float ambientOcclusionGain = 0.0f;
-};
-
 /// @class GlslMaterial
 /// Helper class for GLSL generation and rendering of a material
-class MX_RENDERGLSL_API GlslMaterial
+class MX_RENDERGLSL_API GlslMaterial : public Material
```

Comment:
I really like this approach, where both `GlslMaterial` and `MslMaterial` inherit from a common base class.  One recommendation here would be to name the base class `ShaderMaterial` for consistency with similar base classes such as `ShaderRenderer`, which are designed to be specialized for the needs of individual languages.

Reply:
Thanks for the suggestion. Made this change.